src/pages/Home.jsx

```diff
@@ -6,12 +6,20 @@ import CocktailCard from "../components/CocktailCard"
 const Home = () => {
   const [fetchError, setFetchError] = useState(null)
   const [cocktails, setCocktails] = useState(null)
+  const [orderBy, setOrderBy] = useState('created_at')
```

Comment:
Does this need to be in the state? Is the `orderBy` ever going to change here? If not, a simple variable would do.

I think this should go in a service anyway. It should take a default `orderBy` key.